installer-for-dhs/build.gradle

```diff
@@ -16,10 +16,11 @@ snyk {
 
 dependencies {
     implementation project(":marklogic-data-hub")
-    implementation "com.beust:jcommander:1.72"
-    implementation 'commons-io:commons-io:2.4'
-    implementation "org.slf4j:slf4j-api:1.7.30"
-    implementation 'ch.qos.logback:logback-classic:1.2.3'
+    implementation "com.fasterxml.jackson.core:jackson-databind:2.11.1"
```

Comment:
A few of these have to be brought in because they're no longer API dependencies of the ML Java Client.